### PR TITLE
Change participants and categories form labels

### PR DESCRIPTION
**Pull request by samwilson**
Add extra lines with more user instructions, and link to on-wiki
documentation (via fragment IDs that are already added as
{{anchor}}s on that page).

Bug: T218340

### CODE REVIEW

File: app/Resources/views/events/_categories.html.twig [hunk 1]
```diff
@@ -29,8 +29,17 @@
     </div>
 
     {{ form_start(form, {'method': 'post', 'name': 'categoryForm', 'attr': {'class': 'save-categories-form', 'autocomplete': 'off'}}) }}
-    <p class="save-categories-form__description">{{ msg('categories-description') }}</p>
-    <p>{{ msg('categories-commons-explanation') }}</p>
+    <p class="save-categories-form__description">
+        {{ msg('categories-description') }}
+        <a href="https://meta.wikimedia.org/wiki/Special:MyLanguage/Event_Metrics#category-filtering"
+           title="{{ msg('categories-description-tooltip') }}" class="help-link" target="_blank">
+            <span class="glyphicon glyphicon glyphicon-question-sign"></span>
```
Comment (MusikAnimal) [inline app/Resources/views/events/_categories.html.twig]: two `glyphicon`s

File: i18n/en.json [hunk 2]
```diff
@@ -24,6 +25,8 @@
 	"categories": "Categories",
 	"categories-panel-title": "Categories filter",
 	"categories-description": "Enter categories (with wiki names) to limit metrics to pages in those categories",
+	"categories-description-tooltip": "Learn about Categories filtering",
```
Comment (MusikAnimal) [inline i18n/en.json]: looks like we're missing qqq for this message

File: app/Resources/views/events/_categories.html.twig (hunk 1 above)
Comment (samwilson) [reply app/Resources/views/events/_categories.html.twig]: I've made this mistake before! :)

Fixed.

File: i18n/en.json (hunk 2 above)
Comment (samwilson) [reply i18n/en.json]: Fixed.

**Comment by MusikAnimal** [comment]
I ran the tests locally and they passed.

**Pull request merged**